src/Microsoft.IdentityModel.Tokens/Microsoft.IdentityModel.Tokens.csproj

```diff
@@ -10,10 +10,20 @@
     <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(SrcTargets)</TargetFrameworks>
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">$(SrcStandardTargets)</TargetFrameworks>
     <PackageTags>.NET;Windows;Authentication;Identity;SecurityTokens;Cryptographic operations;Signing;Verifying Signatures;Encryption</PackageTags>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Why do we restrict to 7.2 and below?

Reply:
I picked it up from the MSAL.NET repo / Json PR. 
Builds and tests are also successful with the _latest_ option set.
@MarkZuber - Could you please let me know if the langVersion 7.2 required for some reason?

Reply:
I don't think it is.  We needed more than the default lang version of 7 for a while because we were doing async Task Main() which requires 7.1 or 7.2.  Best way to find out if you need it is to remove this flag and see if your build falls over ;)

Reply:
Great, thanks.